graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/AdminSchemaBuilder.java

```diff
@@ -50,61 +50,6 @@
 
 public class AdminSchemaBuilder {
 
-  private final AuthenticationService authenticationService;
-  private final AuthorizationService authorizationService;
-  private final DataStoreFactory dataStoreFactory;
-
-  public AdminSchemaBuilder(
-      AuthenticationService authenticationService,
-      AuthorizationService authorizationService,
-      DataStoreFactory dataStoreFactory) {
-    this.authenticationService = authenticationService;
-    this.authorizationService = authorizationService;
-    this.dataStoreFactory = dataStoreFactory;
-  }
-
-  public GraphQLSchema build() {
-    return newSchema()
-        .query(QUERY)
-        .mutation(MUTATION)
-        .codeRegistry(
-            newCodeRegistry()
-                .dataFetcher(
-                    coordinates(QUERY, NAMESPACES_QUERY),
-                    new AllNamespacesFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .dataFetcher(
-                    coordinates(QUERY, NAMESPACE_QUERY),
-                    new SingleNamespaceFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .dataFetcher(
-                    coordinates(QUERY, SCHEMA_QUERY),
-                    new SingleSchemaFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .dataFetcher(
-                    coordinates(QUERY, SCHEMA_HISTORY_PER_NAMESPACE_QUERY),
-                    new AllSchemasFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .dataFetcher(
-                    coordinates(MUTATION, CREATE_NAMESPACE_MUTATION),
-                    new CreateNamespaceFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .dataFetcher(
-                    coordinates(MUTATION, DROP_NAMESPACE_MUTATION),
-                    new DropNamespaceFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .dataFetcher(
-                    coordinates(MUTATION, DEPLOY_SCHEMA_MUTATION),
-                    new DeploySchemaFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .dataFetcher(
-                    coordinates(MUTATION, DEPLOY_SCHEMA_FILE_MUTATION),
-                    new DeploySchemaFileFetcher(
-                        authenticationService, authorizationService, dataStoreFactory))
-                .build())
-        .build();
-  }
-
```

Comment:
This is going to be subjective, but I don't think constants should necessarily come first.

When I open a source file I'm not familiar with, what I need is a quick idea of what's going on. Comparing the two versions here:
* if the first thing I see is the `build()` method, it tells me everything I need to know: that what the class builds is a `GraphQLSchema`, and its overall structure. If I'm interested in a particular part, I can use the "jump to definition" feature in my IDE to progressively zoom into it.
* if the first thing I see is a wall of constants, it doesn't tell me how they are being used. I have to sift through some ~400 lines before I get to the meat of the class.

It's not that important so if we want to stick to the convention so be it. But IMHO readability is more important than making Codacy happy.

Same for other constant-related changes in this PR, but they're less significant because they have only a few constants.

Reply:
I also think that we should favor readability over making a tool happy. However, without those changes, we will always get a red build now. We could make exclusions, but I am not sure if we should do it (and how to do it) because the project is OS, and maybe the convention is more important. I can rollback or merge as it is up to you.

Reply:
I prefer we stick to the conventions. Readability will be higher if all the source code looks the same and you always know where to find something (ie constants at the top). Personally the original version of this file feels really weird to me since this format of defining things at the top of the file is very well established.

Reply:
OK, as I said it's not that important. Let's go with the constants first.